salt/modules/postgres.py

```diff
@@ -3109,8 +3122,21 @@ def datadir_init(name,
     locale
         The default locale for new databases
 
+    waldir
+        The transaction log (WAL) directory (default is to keep WAL
+        inside the data directory)
+
+    checksums
+        If True, the cluster will be created with data page checksums.
+
     runas
         The system user the operation should be performed on behalf of
+
+    .. note:
+
+        The ``waldir`` and ``checksum`` options have been added in
+        version XXX. The ``checksum`` option requires PostgreSQL 9.3
```

Comment:
This will be added in `Fluorine`, you can use the `.. versionadded: Fluorine` macro for rst.